src/DetailsView/components/left-nav/getting-started-nav-link.scss

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+@import '../../../common/styles/common.scss';
```

Comment:
Is there a particular reason for this import?

Reply:
Honestly, no. It's not being used so I will remove it. Thanks, John!

Reply:
removed it